Competition2017/tests/competition/subsystems/drive/DriveSubsystemTest.java

```diff
@@ -9,12 +9,6 @@
 public class DriveSubsystemTest extends BaseWPITest {
     @Test
     public void testTankDrive() {
-        DriveSubsystem driveSubsystem = this.injector.getInstance(DriveSubsystem.class);
-        driveSubsystem.tankDrive(1, 1);
-
-        assertEquals(this.mockRobotIO.getPWM(driveSubsystem.leftFrontDrive.getChannel()), 1, 0.001);
-        assertEquals(this.mockRobotIO.getPWM(driveSubsystem.leftRearDrive.getChannel()), 1, 0.001);
-        assertEquals(this.mockRobotIO.getPWM(driveSubsystem.rightFrontDrive.getChannel()), 1, 0.001);
-        assertEquals(this.mockRobotIO.getPWM(driveSubsystem.rightRearDrive.getChannel()), 1, 0.001);
+    
```

Comment:
Let's leave tests to a second PR - I still need to write the basic drive test infra. They'll be coming soon though.